### DISCUSSION TITLE
1.0.8

### DISCUSSION THREAD

**Discussion by FiLinX** [ilyhalight/voice-over-translation#29] [Q&A]
Автор - а не знаешь, почему выводит вот это "Не удалось запросить перевод видео" на этом ролике? - https://www.youtube.com/watch?v=3KEfnPm17KE
версия 1.0.8
раньше вроде выводило всегда только "Перевод займёт больше минуты" и сама кнопка под этой надписью оставалась активной, то есть подождав несколько секунд и повторно нажав уже на эту надпись - почти всегда срабатывал новый запрос и получение "Выполнено", а сейчас при той "Не удалось .." кнопка уже не работает..

пысы:  Эх..  всё оказалось трагичнее  -  расширение вообще перестало работать НА ЛЮБЫХ роликах с ютуба!  Везде теперь пишет "Не удалось запросить перевод видео", даже на тех, что ещё вчера смотрел с переводом через это расширение..
Видимо Яндекс что-то в апи поменял..  Жаль..

Ой, сейчас увидел, что создана issues #27 и наверно эта дискуссия здесь уже лишняя, но как удалить я не знаю..

**Comment by izobretatel9** [comment]
Да, я тоже подтвержадю. Пишет во всех вдео, "Не удалось запросить перевод видео". 
Пользуюсь последним скриптом и Google Chrome последним. 
Раньше было все ок.
